package.json

```diff
@@ -80,6 +80,7 @@
     "is-empty-object": "^1.1.1",
     "js-yaml": "^3.8.4",
     "mocha": "^3.4.2",
+    "mocha-headless-chrome": "^1.8.2",
     "mocha-phantomjs": "^4.0.2",
```

Comment:
`mocha-phantomjs` is not required anymore, right ?

Reply:
Exactly. I'm doing the cleanup later this day.